### ISSUE TITLE
a11y: "dark mode" toggle button cannot be reached and does not have a focus outline

### ISSUE THREAD

**Issue by miclgael** [miclgael/issues.michaelgale.dev#1]
## Steps to reproduce the issue
1. Tab into any web page, e.g. https://www.michaelgale.dev/ 
2. Observe the (unlabelled) Sun/Moon icon in the top right corner of the page is nested between "skip to main content" and "logo/home" links, in terms of DOM order.
3. Advance in tab order once (1x) from the Skip button
4. Observe the unlabelled Sun/Moon icon is not focused 

![Screenshot with a red arrow pointing to the affected portion of UI](https://github.com/miclgael/issues.michaelgale.dev/assets/4301358/3c1cf4b0-cb9f-4351-a973-d25a494b3e1a)

## Expected result

1. A blue focus ring appears around the Sun/Moon icon
2. The ".sr-only" label is announced to screen-reader

## Actual result

Button is skipped entirely

## Device and system info

**What device were you using? (e.g. phone, laptop)**

Desktop PC

**What web browser were you using? (e.g. Chrome, Safari)**

Firefox 102.15.1esr (64-bit) on Windows 11  

**If you were using assistive technology (e.g. VoiceOver, JAWS, NVDA etc.) please list:**

N/A

## Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:

If you have any direct aXe tools documentation please also include this here:

**Comment by miclgael** [comment]
Currently planned to complete by October 13, 2023.

**Comment by miclgael** [comment]
Actually, I had placed the button in the wrong position in source. 

![screenshot showing the new ui after the fix has been made](https://github.com/miclgael/issues.michaelgale.dev/assets/4301358/e451b961-b415-4ed1-9b74-da1e9a361529)


## Changes made

1. Moved the button to be positioned last in the nav
2. Removed `unset: all` from this button, since this is what removed the focus style.
